components/playground/main.go

```diff
@@ -121,11 +119,24 @@ Examples:
   $ tiup playground --pd.config ~/config/pd.toml    # Start a local cluster with specified configuration file,
   $ tiup playground --db.binpath /xx/tidb-server    # Start a local cluster with component binary path`,
 		SilenceUsage: true,
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
Empty function？

Reply:
same as tiup: https://github.com/pingcap/tiup/blob/8c9a182bc0c436e0eb586d1767a2785f43870d2a/cmd/root.go#L52

I don't know how this work actually..., but without setting this Args, after adding subcommand to root cmd, `playground <sub>` , <sub> must be an added subcommand, or 
will meet the error like "unknown subcommand <sub>"